Containerfile

```diff
@@ -0,0 +1,3 @@
+FROM ghcr.io/static-web-server/static-web-server:2.25-alpine
```

Comment:
[GitHub Action Arm Runners](https://github.blog/changelog/2023-10-30-accelerate-your-ci-cd-with-arm-based-hosted-runners-in-github-actions/) are currently in a private beta.  It might be better to disable ARM image builds until then, which would allow us to build and deploy CyberChef inside a multi-stage image?

Then we could start up two actions, one on each architecture, to remove the 30 minute build time!

Reply:
Don't suppose GCHQ has access to the private beta yet 😅? 

I guess since CyberChef does all its operations the browser users won't really suffer a performance penalty from having to run an AMD64 container, I'll make the change to use a multi stage build tomorrow!

Reply:
Maybe this file should also be called `Dockerfile` instead.  Noting that `Docker` aren't intending to support 'Containerfile'. https://github.com/docker/cli/issues/3251

Reply:
or perhaps more projects using Containerfile might incentivise Docker to support it? It is just a case of adding `-f Containerfile` to your build command when running Docker.

Podman falls back to Dockerfile if it cant find a Containerfile, seems only fair for Docker to do the same!

Reply:
Switched over to Docker